apps/web/src/pages/InstructorLedTrainingPage/InstructorLedTrainingPage.tsx

```diff
@@ -0,0 +1,545 @@
+          <span data-h2-font-weight="base(bold)">
+            {formatDate({
+              date: parseDateTimeUtc(trainingOpportunity.trainingStart),
+              formatString: "PPP",
+              intl,
+            })}
+          </span>
+        </>
+      ),
+    } satisfies MetaDataItemProps);
+  }
+
+  if (trainingOpportunity.trainingStart && trainingOpportunity.trainingEnd) {
+    metaData.push({
+      key: `training-dates-${trainingOpportunity.id}`,
+      type: "text",
+      children: (
+        <>
+          {intl.formatMessage({
+            defaultMessage: "Training dates",
+            id: "yK0vlP",
+            description: "Label for training dates meta data",
+          })}
+          {intl.formatMessage(commonMessages.dividingColon)}
+          <span data-h2-font-weight="base(bold)">
+            {formatDate({
+              date: parseDateTimeUtc(trainingOpportunity.trainingStart),
+              formatString: "PPP",
+              intl,
+            })}{" "}
+            {"-"}{" "}
+            {formatDate({
+              date: parseDateTimeUtc(trainingOpportunity.trainingEnd),
+              formatString: "PPP",
+              intl,
+            })}
+          </span>
+        </>
+      ),
+    } satisfies MetaDataItemProps);
+  }
+
+  if (trainingOpportunity.registrationDeadline) {
+    metaData.push({
+      key: `course-format-${trainingOpportunity.id}`,
+      type: "text",
+      children: (
+        <>{getLocalizedName(trainingOpportunity.courseFormat?.label, intl)}</>
+      ),
+    } satisfies MetaDataItemProps);
+  }
+
+  return metaData;
+}
+
+const selectedFilterStyle: Record<string, string> = {
+  mode: "inline",
+  color: "secondary",
+  "data-h2-text-decoration": "base(none)",
+};
+
+const unselectedFilterStyle: Record<string, string> = {
+  mode: "inline",
+  color: "black",
+  "data-h2-font-weight": "base(bold)",
+};
+
+const filterTrainingOpportunities = (
+  trainingOpportunities: TrainingOpportunity[],
+  filterBy?: CourseLanguage.English | CourseLanguage.French | null,
+): TrainingOpportunity[] => {
+  if (!filterBy) {
+    return trainingOpportunities;
+  }
+  const filteredTrainingOpportunities = trainingOpportunities.filter(
+    (trainingOpportunity) => {
+      return trainingOpportunity.courseLanguage?.value === filterBy;
+    },
+  );
+  return filteredTrainingOpportunities;
+};
+
+export const Component = () => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const [trainingOpportunitiesState, setTrainingOpportunitiesState] = useState<
+    TrainingOpportunity[]
+  >([]);
+  const [trainingOpportunitiesFilteredBy, setTrainingOpportunitiesFilteredBy] =
+    useState<CourseLanguage.English | CourseLanguage.French | null>(null);
+
+  const [{ data, fetching }] = useQuery({
+    query: TrainingOpportunitiesPaginated_Query,
+    variables: {
+      where: {
+        hidePassedRegistrationDeadline: true, // Training opportunities past the application deadline do NOT show
```

Comment:
I think you're missing this a/c:
>  Training opportunities cards show (ordered by application deadline, ascending)

Reply:
It's right here 
![image](https://github.com/user-attachments/assets/6c27e178-5eae-4378-984c-6769337e763f)

Reply:
Oh, you're just sorting the page of results, though, right?  This is a server paginated query - you should use the orderBy clause in the query?

Reply:
Doh forgot to use those. Switched to [5f2c04c](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12083/commits/5f2c04c1ff5380300ee0d74683db8043da260318) and [61a6d14](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12083/commits/61a6d14e1619ee545c638ea9b29eb70e5e63594f) through the backend.